docker-compose/Makefile

```diff
@@ -0,0 +1,74 @@
+SHELL := /bin/bash
```

Comment:
Is it possible to print the URLs to use at the end of the make command? I mean:
- Modeling app: http://192.168.1.9.nip.io/activiti-cloud-modeling
- Runtime bundle entry point: http://192.168.1.9.nip.io/rb-my-app
...

Reply:
Can you please add more details what do you mean? In the end of output?

Reply:
yes, when you execute `make all` you have the output:

Reply:
added updates